docs/UserGuide.md

```diff
@@ -166,13 +167,12 @@ This section aims to provide you with in-depth details on Fine\$\$e's unique fea
 **:warning: &nbsp; IMPORTANT &nbsp; :warning:**<br>
 
 The formats of the parameters used in the rest of the document are as follows:
-* `TITLE` and `CATEGORY` should consist of printable ASCII characters, and cannot begin with a space.
+* `TITLE`, `TITLE_KEYPHRASE` and `CATEGORY` should consist of printable ASCII characters.
   * The set of printable ASCII characters consists of alphanumeric characters, space, and several special characters.
     Below is an exhaustive list of printable ASCII characters (excluding line breaks):
-  ```
-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijkl
-mnopqrstuvwxyz !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
-  ```
+    ```
+    0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
+    ```
```

Comment:
This was originally broken up into 2 lines as the final submission has to be a PDF file. When written in a single line, the code block is displayed with a horizontal scrollbar that results in the line being cut off in PDF format.

As a side note, there are multiple other instances in the user guide where code blocks are too long and get truncated after converting to PDF.